tests/thread_msg_avail/Makefile

```diff
@@ -0,0 +1,8 @@
+APPLICATION = thread_msg_avail
+include ../Makefile.tests_common
+
+BOARD_INSUFFICIENT_MEMORY := stm32f0discovery
```

Comment:
I copied it from `thread_msg` but I guess it has insufficient memory there because of 3 threads with printf support, sec ;)